.github/workflows/coding-standards.yml

```diff
@@ -1,14 +1,10 @@
-name: "Coding Standards"
+name: "CS"
```

Comment:
Already discussed in another PR, that's a no for me.

Reply:
I do not prefer it from UX perspective, but ok with me if you do not want to change your decision here :)